<launch>
    <include file="$(find wheelchair_teleop)/launch/wheelchair_control.launch" />

    <!--  smooths inputs from cmd_vel_mux/input/teleop_raw to cmd_vel_mux/input/teleop -->
    <include file="$(find turtlebot_teleop)/launch/includes/velocity_smoother.launch.xml"/>

    <node pkg="wheelchair_teleop" type="wheelchair_teleop_joy_node" name="wheelchair_joystick">
        <param name="scale_angular" value="1.0"/>
        <param name="scale_linear" value="2.0"/>
        <param name="axis_deadman" value="10"/>
        <param name="axis_linear" value="1"/>
        <param name="axis_angular" value="0"/>
        <remap from="wheelchair_joystick/cmd_vel" to="cmd_vel"/>
    </node>

    <node pkg="joy" type="joy_node" name="joystick" />

</launch>
